winevtlog_bench/ansible/windows.yml

```diff
@@ -4,11 +4,22 @@
     td_agent_version: "3.8.0-0"
     eventlog_bencher_version: "v0.4.1.0"
   tasks:
+    - name: Copy local modified version for testing
+      win_copy:
+        src: ./config/td-agent-4.0.50-x64.msi
+        dest: C:\td-agent.msi
+      when: local_package == "yes"
+    - name: Install local td-agent for testing
+      win_package:
+        path="c:\\td-agent.msi"
+        arguments="/q"
+      when: local_package == "yes"
     - name: Install td-agent
       win_package:
         path="http://packages.treasuredata.com.s3.amazonaws.com/3/windows/td-agent-{{ td_agent_version }}-x64.msi"
```

Comment:
off topic: 

path="http://packages.treasuredata.com.s3.amazonaws.com/**3**/windows/td-agent-{{ td_agent_version }}-x64.msi"

**3** should be also configurable...

Reply:
It may be better to handle another PR.

Reply:
Sure.